src/Symfony/Component/Console/Tests/Helper/HelperTest.php

```diff
@@ -21,25 +21,27 @@ public static function formatTimeProvider()
     {
         return [
             [0,      '< 1 sec'],
+            [0.95,   '< 1 sec'],
             [1,      '1 sec'],
             [2,      '2 secs'],
             [59,     '59 secs'],
+            [59.21,  '59 secs'],
             [60,     '1 min'],
-            [61,     '1 min'],
-            [119,    '1 min'],
+            [61,     '1 min, 1 sec'],
+            [119,    '1 min, 59 secs'],
             [120,    '2 mins'],
-            [121,    '2 mins'],
-            [3599,   '59 mins'],
+            [121,    '2 mins, 1 sec'],
+            [3599,   '59 mins, 59 secs'],
             [3600,   '1 hr'],
-            [7199,   '1 hr'],
+            [7199,   '1 hr, 59 mins, 59 secs'],
             [7200,   '2 hrs'],
-            [7201,   '2 hrs'],
-            [86399,  '23 hrs'],
+            [7201,   '2 hrs, 1 sec'],
+            [86399,  '23 hrs, 59 mins, 59 secs'],
             [86400,  '1 day'],
-            [86401,  '1 day'],
-            [172799, '1 day'],
+            [86401,  '1 day, 1 sec'],
+            [172799, '1 day, 23 hrs, 59 mins, 59 secs'],
```

Comment:
Do we need precision to the second when the duration is in days? 2 units should be enough.

```suggestion
            [172799, '1 day, 23 hrs'],
```